internal/controller/nim/account_controller.go

```diff
@@ -69,6 +69,7 @@ var (
 func (r *AccountReconciler) SetupWithManager(mgr ctrl.Manager, ctx context.Context) error {
 	// TODO: Copied from original main.go... Should it be FromContext?
```

Comment:
yes we are using log fromContext if there are no issues with it.  Do you have any issues to use the way?

Reply:
@Jooho About this TODO comment, I can't seem to find anywhere else we use `FromContext` at setup stage.

Reply:
Also, we use the SignalHandler context; see [here](https://github.com/opendatahub-io/odh-model-controller/blob/incubating/cmd/main.go#L170). I don't think it has a logger at the point we create it.

Reply:
In the main.go file, the way the reconciler is registered for NIM is somewhat different from the others. Is there a specific reason for this? It would be better to keep the structure consistent with the rest if possible. current structure in odh-model-controller missed to print out the error when reconciller failed(I need to follow up this) but the error log corrected in the main.go level(https://github.com/opendatahub-io/odh-model-controller/blob/incubating/cmd/main.go#L147)
However, it is not related to this pr so we can discuss it later.

Reply:
Hi @Jooho 

I switched to using the setup logger here: https://github.com/opendatahub-io/odh-model-controller/pull/391.